midp/midp.js

```diff
@@ -927,7 +927,17 @@ var MIDP = (function() {
       var nativeEventQueue = nativeEventQueues[isolateId];
 
       if (nativeEventQueue.length !== 0) {
-        copyEvent(nativeEventQueue.shift(), nativeEvent);
+        var ev = nativeEventQueue.shift();
+        if (ev.type === PEN_EVENT
+         && ev.intParam1 === DRAGGED) {
```

Comment:
Nit: the indentation here and in the *while* is off, we usually do:
```
if (aFirstCondition &&
    aSecondCondition) {
```

In this particular case, the two conditions could fit in one line.